src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -56,14 +56,13 @@ public function onKernelRequest(RequestEvent $event)
             return;
         }
 
-        $session = null;
         $request = $event->getRequest();
         if (!$request->hasSession()) {
             $sess = null;
```

Comment:
I wonder if this shouldn't have been `session` here and below?

Otherwise, I don't get the purpose of the code at line 65

Reply:
I took a look the code and they have never been mixed. IMO The internal `$sess` acts as a memoized/cached version. However I think that edit was introduced in #38910.  However, I'm not a Symfony's internals expert.

Reply:
indeed, this is not related. the `$session` here was introduced at a time we performed several operations in the if/else bloc and forgot to remover it in https://github.com/symfony/symfony/commit/b9ca866bdd3c92f2990f2ce7ef9914020ded21d4